src/feedback/components/ActivityFeedback.vue

```diff
@@ -52,11 +75,13 @@
           </div>
         </div>
         <FeedbackForm
+          v-if="select && select.activityType"
           :value="feedbackDefault"
           :status="saveStatus"
           :is-bike-kitchen="isBikeKitchen"
           :is-general-purpose="isGeneralPurpose"
```

Comment:
Can remove this theme variables from everywhere?

Reply:
Not everywhere yet (setting css class for layout for background image I guess, and still using a locale override), but some removed...